src/pages/api/webhooks/image-scan-result.ts

```diff
@@ -280,15 +280,16 @@ async function handleSuccess({ id, tags: incomingTags = [], source, context }: B
     const inappropriate = includesInappropriate(prompt, nsfw);
     if (inappropriate !== false) reviewKey = inappropriate;
     if (!reviewKey && nsfw) {
-      const [{ poi }] = await dbWrite.$queryRaw<{ poi: boolean }[]>`
+      const [{ poi, minor }] = await dbWrite.$queryRaw<{ poi: boolean; minor: boolean }[]>`
         WITH to_check AS (
           -- Check based on associated resources
           SELECT
-            COUNT(*) > 0 "poi"
+            COUNT(*) > 0 "poi",
+            COUNT(*) FILTER (WHERE m.minor) > 0 "minor"
           FROM "ImageResource" ir
           JOIN "ModelVersion" mv ON ir."modelVersionId" = mv.id
           JOIN "Model" m ON m.id = mv."modelId"
-          WHERE ir."imageId" = ${image.id} AND m.poi
+          WHERE ir."imageId" = ${image.id}
```

Comment:
If we remove the `AND` here, I guess it'll count all resources as POI 🤔 we probably need a filter for "poi" as well